src/Phinx/Console/Command/AbstractCommand.php

```diff
@@ -318,6 +318,10 @@ protected function loadManager(InputInterface $input, OutputInterface $output)
     {
         if ($this->getManager() === null) {
             $manager = new Manager($this->getConfig(), $input, $output);
+            $container = $this->config->getContainer();
```

Comment:
This might have caused a regression in master

> Error: [Error] Call to a member function getContainer() on null in vendor/robmorgan/phinx/src/Phinx/Console/Command/AbstractCommand.php on line 321

Reply:
For some reason this seems to fix it: https://github.com/cakephp/phinx/commit/18202cdb700cac236436dc1e0a934faf2dcf0b04

Reply:
Oh dear. Sorry about that! I'm a bit confused as to how $this->config is null when above $this->getConfig() is passing $this->config into new Manager which expects an object which is not null. Could you give me steps to reproduce?

Reply:
That is very weird. Can't see why that would make a difference.

Reply:
Yeah, I also dont understand it. The changed code worked though for now.

Reply:
Is this in phinx proper or in a wrapper of it? How does `$this->config` be null, yet `getConfig()`, which simply returns `$this->config` return something that is not null?

Reply:
It is in the Migrations wrapper ( https://github.com/cakephp/migrations )

Reply:
Perhaps ->getConfig() is being overridden in a child class